integration/app/test-demo/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const app = 'meeting';
+
+module.exports = {
+  mode: 'development',
+  entry: ['./src/index.tsx'],
+  devtool: 'inline-source-map',
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/
+      },
+    ]
+  },
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+  },
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: `${app}-bundle.js`,
+    publicPath: '/',
+    libraryTarget: 'var',
+    library: `app_${app}`
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      inlineSource: '.(js|css)$',
+      template: __dirname + `/public/${app}.html`,
+      filename: __dirname + `/dist/${app}.html`,
+      inject: 'head'
+    }),
+  ],
+  devServer: {
+    proxy: {
+      '/': {
+        target: 'http://127.0.0.1:8080',
+        bypass: function (req, _res, _proxyOptions) {
+          if (req.headers.accept.indexOf('html') !== -1) {
+            console.log('Skipping proxy for browser request.');
+            return `/${app}.html`;
+          }
+        }
+      }
+    },
+    hot: false,
+    port: 9000,
+  }
```

Comment:
can we add the open:true option? I like it. It opens the app for you when its ready. https://webpack.js.org/configuration/dev-server/#devserveropen

Reply:
I don't think we need this. This is a demo for integration testing. WebDriver needs to load the demo URL anyway.